Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebsiteDatastore.mm

```diff
@@ -527,6 +527,63 @@ HTTPServer server([connectionCount = 0] (Connection connection) mutable {
     EXPECT_FALSE([fileManager fileExistsAtPath:generalStorageDirectory.get().path]);
 }
 
+TEST(WKWebsiteDataStore, RemoveDataStoreWithIdentifierRemoveCredentials)
+{
+    // FIXME: we should use persistent credential for test after rdar://100722784 is in build.
+    usePersistentCredentialStorage = false;
```

Comment:
Don't we need to set done to false here? 

In this test, it looks like to reset done to false after it becomes true, however, it looks like the previous test doesn't.